src/manager/node.c

```diff
@@ -799,31 +884,33 @@ static int node_disconnected(UNUSED sd_bus_message *message, void *userdata, UNU
                 }
 
                 usubs->loaded = false;
+
                 UnitSubscription *usub = NULL;
```

Comment:
Hmm, this only sends the virtual events for non-wildcard subscriptions. Also, we don't even track the last state for such subscriptions, so we can't sent any state change event...

Reply:
As we discussed, I changed this so that virtual `UnitNew` and `UnitRemoved` (also StateChanged) signals are emitted. This way the monitor gets notified if, for example, the node disconnects and reconnects, and can act upon it (e.g. query current state of special units).